coding_systems/ctv3/README

```diff
@@ -6,5 +6,24 @@ CTV3 is a retired system of clinical terms.
 It is the successor to Read v2, and is replaced by SNOMED CT.  (The "CT" stands for
 "Clinical Terms" in both CTV3 and SNOMED CT.)
 
-No new CTV3 codes are published, although sometimes TPP have published new "Y codes",
-which look like CTV3 codes.
+No new CTV3 codes are published, although sometimes TPP have published new "Y codes", which look like CTV3 codes.
+
+The last release of CTV3 was on 1st Aug 2018.  This data is no longer available to
+download online, but can be retrieved from the zip file on dokku1: `nhs_readctv3_25.0.0_20180401000001.zip`.
```

Comment:
Isn't dokku1 going away?  Should we move this somewhere more permanent?

Reply:
yes, all refs to dokku1 should be dokku3 now - there are a few other places in docs to update too